aio/src/app/shared/custom-icon-registry.ts

```diff
@@ -57,11 +57,13 @@ export class CustomIconRegistry extends MatIconRegistry {
   }
 
   private loadSvgElements(svgIcons: SvgIconInfo[]) {
-    const div = document.createElement('DIV');
     svgIcons.forEach(icon => {
       const ns = icon.namespace || DEFAULT_NS;
       const nsIconMap = this.preloadedSvgElements[ns] || (this.preloadedSvgElements[ns] = {});
 
+      // this change is fix for IE11 to show the SVG correctly
```

Comment:
It is not clear what the change is by looking at the comment. How about making more clear (without having to look at the code histiry) with something like:

```suggestion
      // Creating a new `<div>` per icon is necessary for the SVGs to work correctly in IE11.
```